crates/catalog/rest/src/catalog.rs

```diff
@@ -745,10 +749,86 @@ impl Catalog for RestCatalog {
         _table_ident: &TableIdent,
```

Comment:
the `_` prefix is for unused variables, we should remove it since now it's implemented